inclusive-design/wecount.inclusivedesign.ca
# fix: remove bootstrap vue

## Description

Removes Bootstrap Vue dependency.

## Steps to test

1. Visit all routes.

**Expected behavior:** All routes work without the use of Bootstrap Vue components.

## Additional information

Not applicable.

## Related issues

- Resolves #49.
